app/controllers/admin_users_sessions_controller.rb

```diff
@@ -24,4 +24,12 @@ def create
 
     redirect_to user_path(@user_to_login_as)
   end
+
+  def destroy
```

Comment:
Worth adding some basic specs to this.

- Check that we're now logged in as the correct admin
- Check that we redirected to the right place